sites/public/styles/overrides.scss

```diff
@@ -56,9 +56,41 @@
           border: none;
         }
 
+        .navbar-dropdown-container {
+          margin-left: -144px;
+          width: auto;
+          .navbar-dropdown {
+            .navbar-dropdown-item-container {
+              display: flex;
+            }
+            .navbar-dropdown-item {
+              @apply border-t;
+              @apply border-gray-450;
+              @apply flex;
+              @apply flex-1;
+              @apply items-center;
+              @apply normal-case;
+              @apply pt-3;
+              @apply pb-3;
+              @apply pl-3;
+              @apply pr-3;
+              @apply text-gray-750;
+              font-size: 0.7rem;
```

Comment:
Are you sure we want .7rem here? 16 * .7 = 11.2. It's a little weird to do default fractional font sizes. Maybe do .75 (12px)?

Reply:
Done.